hack/install.sh

```diff
@@ -19,5 +19,6 @@ dump_state.setup
 scale_up_workers
 create_namespaces "${SYSTEM_NAMESPACES[@]}"
 
+install_certmanager
```

Comment:
we always need cert-manager now for Serving tests, so we can install it in any make target

Reply:
For Serving we now always have the one test making sure TLS stuff works, so we always need it. Also it has to be there first, so we can populate CRDs and the trust bundle upfront.